protocol/protobuf/application_metadata_message.proto

```diff
@@ -13,22 +13,15 @@ message ApplicationMetadataMessage {
   Type type = 3;
 
   enum Type {
-    reserved 14;
-    reserved "SYNC_INSTALLATION_PUBLIC_CHAT";
     UNKNOWN = 0;
     CHAT_MESSAGE = 1;
     CONTACT_UPDATE = 2;
     MEMBERSHIP_UPDATE_MESSAGE = 3;
     SYNC_PAIR_INSTALLATION = 4;
-    DEPRECATED_SYNC_INSTALLATION = 5 [deprecated=true];
-    REQUEST_ADDRESS_FOR_TRANSACTION = 6 [deprecated = true];
-    ACCEPT_REQUEST_ADDRESS_FOR_TRANSACTION = 7 [deprecated = true];
-    DECLINE_REQUEST_ADDRESS_FOR_TRANSACTION = 8 [deprecated = true];
-    REQUEST_TRANSACTION = 9 [deprecated = true];
-    SEND_TRANSACTION = 10 [deprecated = true];
-    DECLINE_REQUEST_TRANSACTION = 11 [deprecated = true];
+    reserved 5, 6, 7, 8, 9, 10, 11;
```

Comment:
Is `reserved` better that `deprecated`? 🤔 
And if we keep reserved, then I believe we should also reserve the name, as it was done for some of the enums here.